javascript/generate_protos.sh

```diff
@@ -2,4 +2,4 @@
 set -e
 set -x
 
-PATH=node_modules/protoc-gen-ts/bin/:$PATH protoc -I=../proto --ts_out=dist cacheclient.proto controlclient.proto
+PATH=node_modules/protoc-gen-ts/bin/:$PATH protoc -I=../proto -I=/usr/local/include --ts_out=dist cacheclient.proto controlclient.proto
```

Comment:
do we want both `-I ../proto` and ` -I /usr/local/include`?

Reply:
/usr/local/include puts protocol buffers typedefs on the find path. I think that was always the intent, no? I don't care that much, it's just a bomb for $future_dev if they add a `protobuf` typed thing into our api ever. 🤷